src/@next/Select/Select.tsx

```diff
@@ -136,14 +145,21 @@ export const Select = ({
     setPopoverActive(!popoverActive);
   };
 
+  useEffect(
+    function syncInputValueFromSearchableProps() {
+      setInputValue(searchableProps?.inputValue);
+    },
+    [searchableProps?.inputValue]
+  );
+
```

Comment:
can you share what’s the bug that this useEffect is fixing?

Reply:
Sure! you can refer to this video
https://www.loom.com/share/52216222926c4bd98e67d7188d3acdd0?sid=fc28029c-67ef-40cd-a2dc-0481a8b8568b

the context is when user clears the country fields, would auto-clear the city field at the same time.
But you can see the video, if I search any cities, then change the country, the city field will displaying the keyword that I used previously.
I found that it's because select component doesn't sync up the `searchableProps?.inputValue`